### PR TITLE
[Fix] Remove usernames and Wiki Challenge Solution Template reference in English guide articles

### PR DESCRIPTION
**Pull request by RandellDawson**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

This PR came about based on noticing a Github username was added in another PR (#34858). Please read the comments on this other PR for background on why my PR was created.

Basically, it removes 3 types of content on applicable curriculum challenge articles:

references to Add your username at the bottom of the article
references to the Wiki Challenge Solution Template - because we no longer require a specific template
references to Thanks @username

### CODE REVIEW

**Review by thecodingaviator — CHANGES_REQUESTED** [review]
These 4 files still have the text:

image

**Comment by RandellDawson** [comment]
@thecodingaviator Not sure how I missed those 4. Oh well, I have taken care of those now too.

Thanks for the review.

**Review by thecodingaviator — APPROVED** [review]
LGTM! 👍

File: guide/english/certifications/javascript-algorithms-and-data-structures/es6/set-default-parameters-for-your-functions/index.md [hunk 1]
```diff
@@ -64,7 +64,3 @@ Relevant Links:
 * :warning: DO NOT add solutions that are similar to any existing solutions. If you think it is similar but better, then try to merge (or replace) the existing similar solution.
```
Comment (ojeytonwilliams) [inline guide/english/certifications/javascript-algorithms-and-data-structures/es6/set-default-parameters-for-your-functions/index.md]: See above.

**Review by ojeytonwilliams — COMMENTED** [review]
The PR itself LGTM, however I noticed a few problems with the files it addresses and I wondered if they should be dealt with in this PR or not.   See below for the problems.

File: guide/english/certifications/javascript-algorithms-and-data-structures/basic-javascript/nesting-for-loops/index.md [hunk 2]
```diff
@@ -93,9 +93,4 @@ multiplyAll([[1,2],[3,4],[5,6,7]]);
 <li>:warning: <strong>DO NOT</strong> add solutions that are similar to any existing solutions. If you think it is similar but better, then try to merge (or replace) the existing similar solution.</li>
```
[the remark this answers is not in the document]
Comment (RandellDawson) [reply guide/english/certifications/javascript-algorithms-and-data-structures/basic-javascript/nesting-for-loops/index.md]: I think this issue should be handled on a separate PR which captures all such occurrences instead of just the ones on these 58 files, since it was not something I changed or caused on this PR.

It is kind of like if you found typos in other sections of these files.  That would be beyond this PR's scope.

On a related note, I created [an issue](https://github.com/freeCodeCamp/freeCodeCamp/issues/35453) a while back for the sole purpose of coming up with what the MVP of a curriculum-related Guide article should even look like.  I personally want to simplify all of these and move away from all of these icons/images.

Also, I have created [another issue](https://github.com/freeCodeCamp/freeCodeCamp/issues/35479) (and PR) related to removing all the repl.it and codepen.io links from all of the curriculum-related Guide articles.  Now that we are getting the number of open Guide articles down, we can start doing these kind of mass-change PRs to clean up these articles.

[the remark this answers is not in the document]
Comment (ojeytonwilliams) [reply guide/english/certifications/javascript-algorithms-and-data-structures/basic-javascript/nesting-for-loops/index.md]: Yeah, I suspected as much.

I saw those issues and am all in favour.  In the mean time, I'll see about creating a PR for these bugs.  Even though the emojis are not long for this world, it's an easy fix. It's only these 3 files as far as I can see.

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1:

File: guide/english/certifications/javascript-algorithms-and-data-structures/basic-javascript/nesting-for-loops/index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (RandellDawson) [reply guide/english/certifications/javascript-algorithms-and-data-structures/basic-javascript/nesting-for-loops/index.md]: @ojeytonwilliams FYI - you are free to make the changes regarding emoji fixes on this PR if you want.  I just did not want to hold it up for that.  You can add another commit which does this.  I have no issues with that.

[the remark this answers is not in the document]
Comment (ojeytonwilliams) [reply guide/english/certifications/javascript-algorithms-and-data-structures/basic-javascript/nesting-for-loops/index.md]: @RandellDawson No, no it's fine - I'll create a separate PR.  You're right that it's not really connected, it just happens in the same files.

Also, I'm about to sign off for the day, so I'd only end up needlessly delaying this PR.

[the remark this answers is not in the document]
Comment (RandellDawson) [reply guide/english/certifications/javascript-algorithms-and-data-structures/basic-javascript/nesting-for-loops/index.md]: OK.  I think @thecodingaviator is going to merge it soon.

**Comment by thecodingaviator** [comment]
@RandellDawson Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 📝
